apps/session-manager/src/services/authentication-lock.ts

```diff
@@ -0,0 +1,17 @@
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import * as RTE from "fp-ts/ReaderTaskEither";
+import { flow } from "fp-ts/lib/function";
+import * as TE from "fp-ts/TaskEither";
+import * as ROA from "fp-ts/ReadonlyArray";
+import { LockedProfileRepo } from "../repositories";
+import { LockUserAuthenticationDeps } from "../repositories/locked-profile";
+
+type AuthLockServiceDeps = LockUserAuthenticationDeps;
```

Comment:
I think our new `index` file management inside some folders is getting out of hands with the increment of types and variables exported. we know that the LSP will surely automatically import the type or variable directly instead of passing through our alias exported in the `index.ts` file binded in the folder. we should talk about this and pick a decision
```suggestion
import { LockedProfileRepo } from "../repositories";

type AuthLockServiceDeps = LockedProfileRepo.LockUserAuthenticationDeps;
```